src/game/client/swarm/c_asw_weapon.h

```diff
@@ -14,6 +14,10 @@ class C_ASW_Marine;
 class CASW_WeaponInfo;
 class C_BreakableProp;
 
+static bool g_bLaserIndexInUse[8]{}; //LS INDEX SIZE 8
```

Comment:
don't define a naked static variable in a header - this causes each file that includes the header to have a separate instance of the variable

you may want extern here and then the same variable without extern in a cpp file